Sources/FTPropertyWrappers/Keychain - Support/KeychainCoding.swift

```diff
@@ -0,0 +1,371 @@
+import Foundation
```

Comment:
Can we separate encoding and decoding into separate types? To respect single-responsibility principle.